dataclients/kube/definitions.go

```diff
@@ -0,0 +1,115 @@
+package kube
```

Comment:
same here: I would propose using the long form "kubernetes" everywhere (we don't need to save any ASCII chars, do we?)